server/controllers/upload.js

```diff
@@ -0,0 +1,23 @@
+const Profile = require('../models/Profile')
+const asyncHandler = require("express-async-handler");
+
+// @route POST /upload/profile
+// @desc Update user's profile photo
+// @access Public
+exports.updateProfilePhoto = asyncHandler(async (req, res, next) => {
+  const userId = req.user.id;
+  let profile;
+
+  if (!req.file) {
+    res.status(400).json({ error: "Couldn't upload photo" });
+  } else {
+    if (userId) {
```

Comment:
I don't think we'll need this 'if' clause - only logged in users who are authenticated will be able to access this route (meaning they will have a user id)